{"uid":"dd3cc6ee3a64b87b","name":"Create a tag under repo","fullName":"tests.test_tag#test_create_tag_under_repo","historyId":"8ec1222f5e3cf07f7fe697da8965089e","time":{"start":1751888105100,"stop":1751888105100,"duration":0},"status":"broken","statusMessage":"ValueError: must be set environment variables.","statusTrace":"@pytest.fixture(scope=\"session\")\n    def config():\n        base_url = os.getenv(\"GITCODE_BASE_URL\")\n        owner = os.getenv(\"GITCODE_PERSONAL_OWNER\")\n        token = os.getenv(\"GITCODE_PERSONAL_ACCESS_TOKEN\")\n        if not base_url or not owner or not token:\n>           raise ValueError(\"must be set environment variables.\")\nE           ValueError: must be set environment variables.\n\ntests/conftest.py:18: ValueError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"config","time":{"start":1751888105100,"stop":1751888105100,"duration":0},"status":"broken","statusMessage":"ValueError: must be set environment variables.\n","statusTrace":"  File \"/home/runner/work/pytest-template/pytest-template/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/home/runner/work/pytest-template/pytest-template/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"/home/runner/work/pytest-template/pytest-template/.venv/lib/python3.13/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/pytest-template/pytest-template/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/pytest-template/pytest-template/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/pytest-template/pytest-template/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 929, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/pytest-template/pytest-template/tests/conftest.py\", line 18, in config\n    raise ValueError(\"must be set environment variables.\")\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true}],"afterStages":[],"labels":[{"name":"tag","value":"API"},{"name":"owner","value":"chaocw"},{"name":"suite","value":"Tag"},{"name":"severity","value":"critical"},{"name":"parentSuite","value":"tests"},{"name":"host","value":"pkrvmbietmlfzoi"},{"name":"thread","value":"2396-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_tag"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"critical","owner":"chaocw","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["API"]},"source":"dd3cc6ee3a64b87b.json","parameterValues":[]}